code/game/gamemodes/gang/gang_items.dm

```diff
@@ -312,36 +568,147 @@
 
-/datum/gang_item/equipment/implant_breaker/spawn_item(mob/living/carbon/user, datum/gang/gang, obj/item/device/gangtool/gangtool)
+/datum/gang_item/equipment/recruiter/spawn_item(mob/living/carbon/user, datum/gang/gang, obj/item/device/gangtool/gangtool)
 	if(item_path)
 		var/obj/item/O = new item_path(user.loc, gang) //we need to override this whole proc for this one argument
 		user.put_in_hands(O)
 	if(spawn_msg)
 		to_chat(user, spawn_msg)
-		
+
 /datum/gang_item/equipment/wetwork_boots
 	name = "Wetwork boots"
 	id = "wetwork"
 	cost = 20
 	item_path = /obj/item/clothing/shoes/combat/gang
-	
+
+//////// REVIVIFICATION SERUM //////////
+
+/datum/gang_item/equipment/reviver
+	name = "Outlawed Reviver Serum"
+	id = "reviver"
+	cost = 50
+	item_path = /obj/item/weapon/reviver
+
+/datum/gang_item/equipment/reviver/get_cost(mob/living/carbon/user, datum/gang/gang, obj/item/device/gangtool/gangtool)
+	cost = 10 + gang.gangsters.len * 2
+	return cost
+
+/obj/item/weapon/reviver
+	name = "outlawed revivification serum"
+	desc = "Banned due to side effects of extreme rage, reduced intelligence, and violence. For gangs, that's just a fringe benefit."
+	icon = 'icons/obj/items.dmi'
+	icon_state = "implanter1"
+	item_state = "syringe_0"
+	throw_speed = 3
+	throw_range = 5
+	w_class = WEIGHT_CLASS_SMALL
+	origin_tech = "materials=2;biotech=5"
+	materials = list(MAT_METAL=600, MAT_GLASS=200)
+
+
+/obj/item/weapon/reviver/attack(mob/living/carbon/human/H, mob/user)
+	if(!ishuman(H))
+		return
+	user.visible_message("<span class='warning'>[user] begins inject [H] with [src].</span>", "<span class='warning'>You begin to inject [H] with [src]...</span>")
+	var/total_burn	= 0
+	var/total_brute	= 0
+	H.notify_ghost_cloning("You're being injected with a revivification serum!")
+	if(do_after(user, 30, target = H))
+		if(H.stat == DEAD)
+			H.visible_message("<span class='warning'>[H]'s body thrashes violently.")
+			playsound(get_turf(src), "bodyfall", 50, 1)
+			total_brute = H.getBruteLoss()
+			total_burn = H.getFireLoss()
+			if (H.suiciding || (H.disabilities & NOCLONE) || !H.getorgan(/obj/item/organ/heart) || !H.getorgan(/obj/item/organ/brain))
+				H.visible_message("<span class='warning'>[H]'s body falls still again, they're gone for good.")
+				return
+			var/overall_damage = total_brute + total_burn + H.getToxLoss() + H.getOxyLoss()
+			var/mobhealth = H.health
+			H.adjustOxyLoss((mobhealth - HALFWAYCRIT) * (H.getOxyLoss() / overall_damage), 0)
+			H.adjustToxLoss((mobhealth - HALFWAYCRIT) * (H.getToxLoss() / overall_damage), 0)
+			H.adjustFireLoss((mobhealth - HALFWAYCRIT) * (total_burn / overall_damage), 0)
+			H.adjustBruteLoss((mobhealth - HALFWAYCRIT) * (total_brute / overall_damage), 0)
+			H.updatehealth()
+			H.set_heartattack(FALSE)
+			H.grab_ghost()
+			H.revive()
+			H.emote("gasp")
+			H.setBrainLoss(70)
+			add_logs(user, H, "revived", src)
+			qdel(src)
```

Comment:
I wasn't really impressed with the gang implanter's version that leaves behind a bunch of needles but I suppose its worth doing.